src/erk/cli/github_parsing.py

```diff
@@ -79,24 +76,20 @@ def parse_pr_identifier(identifier: str) -> int:
         42
         >>> parse_pr_identifier("https://github.com/owner/repo/pull/123")
         123
-        >>> parse_pr_identifier("https://github.company.com/org/repo/pull/888")
-        888
     """
     # Plain number (handles leading zeros like "0042" -> 42)
     if identifier.isdigit():
         return int(identifier)
 
-    # Try strict github.com URL first
+    # Try strict github.com URL (PR or issue URL)
     pr_number = parse_pr_number_from_url(identifier)
     if pr_number is not None:
         return pr_number
 
-    # Fallback: permissive pattern for GitHub Enterprise and other URLs
-    # Matches /pull/123 or /issues/123 or plain 123 at end of string
-    pattern = r"(?:(?:pull|issues)/)?(\d+)(?:[?#].*)?$"
-    match = re.search(pattern, identifier)
-    if match:
-        return int(match.group(1))
+    # Also accept /issues/ URLs (same repository reference pattern)
+    issue_number = parse_issue_number_from_url(identifier)
+    if issue_number is not None:
+        return issue_number
```

Comment:
i don't think we should accept issues. can you give a good reason why?

Reply:
Removed /issues/ URL support. You're right - there's no good reason to accept /issues/ URLs when parsing PR references. The old behavior was a side effect of the permissive fallback regex. Now only plain numbers and /pull/ URLs are accepted.

_Addressed via `/erk:pr-address` at 2025-12-30 10:46 PST_